tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -225,23 +227,47 @@ install_kernel_experimental() {
 	DESTDIR="${destdir}" PREFIX="${prefix}" "${kernel_builder}" -f -b experimental -v ${kernel_version}
 }
 
-# Install static qemu asset
-install_qemu() {
-	export qemu_repo="$(yq r $versions_yaml assets.hypervisor.qemu.url)"
-	export qemu_version="$(yq r $versions_yaml assets.hypervisor.qemu.version)"
+install_qemu_helper() {
+	local qemu_repo_yaml_path="${1}"
+	local qemu_version_yaml_path="${2}"
+	local qemu_name="${3}"
+	local builder="${4}"
+
+	export qemu_repo="$(yq r $versions_yaml ${qemu_repo_yaml_path})"
+	export qemu_version="$(yq r $versions_yaml ${qemu_version_yaml_path})"
 
 	install_cached_tarball_component \
-		"QEMU" \
-		"${jenkins_url}/job/kata-containers-main-qemu-$(uname -m)/${cached_artifacts_path}" \
+		"${qemu_name}" \
+		"${jenkins_url}/job/kata-containers-main-${qemu_name}-$(uname -m)/${cached_artifacts_path}" \
 		"${qemu_version}-$(calc_qemu_files_sha256sum)" \
 		"$(get_qemu_image_name)" \
 		"${final_tarball_name}" \
 		"${final_tarball_path}" \
 		&& return 0
 
-	info "build static qemu"
-	"${qemu_builder}"
-	tar xvf "${builddir}/kata-static-qemu.tar.gz" -C "${destdir}"
+	info "build static ${qemu_name}"
+	"${builder}"
+	tar xvf "${builddir}/kata-static-${qemu_name}.tar.gz" -C "${destdir}"
+}
+
+# Install static qemu asset
+install_qemu() {
+	install_qemu_helper \
+		"assets.hypervisor.qemu.url" \
+		"assets.hypervisor.qemu.version" \
+		"qemu" \
+		"${qemu_builder}"
+}
+
+install_qemu_tdx_experimental() {
+	export qemu_suffix="tdx-experimental"
+	export qemu_tarball_name="kata-static-qemu-tdx-experimental.tar.gz"
```

Comment:
Nit: `export qemu_tarball_name="kata-static-qemu-${qemu_suffix}.tar.gz"`

and on line 269: `"qemu-${qemu_suffix}"`